### PR TITLE
Update ui design for the app

### PR DESCRIPTION
**Pull request by zengzihui**
- update the color scheme
- update the ui designs

### CODE REVIEW

**Review by KhoonSun47 — APPROVED** [review]
LGTM! Very nice UI!

File: src/main/resources/view/HouseListCard.fxml [hunk 1]
```diff
@@ -5,15 +5,54 @@
...
+<!--    <Label fx:id="block" styleClass="small-label" text="\$block">-->
+<!--        <padding>-->
+<!--            <Insets top="5"  bottom="5"/>-->
+<!--        </padding>-->
+<!--    </Label>-->
+<!--    <Label fx:id="level" styleClass="small-label" text="\$level">-->
+<!--        <padding>-->
+<!--            <Insets top="5"  bottom="5"/>-->
+<!--        </padding>-->
+<!--    </Label>-->
```
Comment (lokidoki102) [inline src/main/resources/view/HouseListCard.fxml:51]: I think can just remove the commented block of codes

**Review by lokidoki102 — APPROVED** [review]
Very nice UI!

**Comment by felixchanyy** [comment]
Nice Ui!

**Pull request merged**